website_sale_stock_provisioning_time/models/product_template.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    provisioning_time_in_workdays = fields.Integer(
+        string="Provisioning time in workdays"
```

Comment:
workdays --> work days I think.

Reply:
It seems both to be ok:

* https://dict.leo.org/englisch-deutsch/arbeitstage
* https://www.dict.cc/?s=arbeitstage
* https://www.deepl.com/translator#de/en/arbeitstage